.github/workflows/e2e-full.yml

```diff
@@ -23,35 +24,27 @@ jobs:
             globalnet: 'globalnet'
           - ovn: 'ovn'
             cable_driver: 'wireguard'
+        include:
+          - external_net: 'external-net'
+          - external_net: 'external-net'
+            globalnet: 'globalnet'
```

Comment:
As I read https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs#example-including-new-combinations , I understand that only the combinations of ('external-net') and ('external-net', 'globalnet') is added by this include lines. And these are currently defined in e2e-external job, and are merged to this e2e job. Looks good to me.